### PR TITLE
Define AWS 9.2.2 WIP

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/9714

### CODE REVIEW

**Comment by sslavic** [comment]
@giantswarm/team-ludacris maybe you're interested the most in having this release with app updates shipped, adidas reported issue (see [here](https://gigantic.slack.com/archives/C6L8J93N0/p1585075236382200)) when upgrading from 9.0.0 to 9.2.1 that coredns was down relatively long due to no PDB; aws 9.2.1 release has coredns-app 1.1.3 while PDB was already added in newer coredns-app 1.1.6, and even newer 1.1.8 is available.

**Comment by MarcelMue** [comment]
@sslavic Okay - we had the same issue testing upgrades from `9.0.1` to `9.2.1` where the cordens helm release was stuck in `PENDING`.

**Comment by sslavic** [comment]
@MarcelMue helm releases stuck in pending on k8s 1.15 to 1.16 upgrade is a different issue (see [here](https://github.com/giantswarm/giantswarm/issues/9705)) from PDB missing.

File: aws.yaml [hunk 1]
```diff
@@ -209,6 +209,55 @@
...
+      version: 0.12.1
+    - app: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.4
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - app: kube-state-metrics
+      componentVersion: 1.9.2
+      version: 1.0.4
```
Comment (sslavic) [inline aws.yaml]:
```suggestion
    - app: kube-state-metrics
      componentVersion: 1.9.5
      version: 1.0.5
```

File: aws.yaml [hunk 2]
```diff
@@ -209,6 +209,55 @@
       version: 3.9.1
     - name: etcd
       version: 3.3.15
+- version: 9.2.2
+  state: wip
+  date: 2020-03-23T12:00:00Z
```
Comment (sslavic) [inline aws.yaml]:
```suggestion
  date: 2020-03-27T12:00:00Z
```

**Pull request merged**